controllers/validations.go

```diff
@@ -143,8 +143,13 @@ func checkStorageClusterPeerStatus(ctx context.Context, client client.Client, lo
 		}
 		logger.Info("StorageClusterPeer ManifestWork has reached Applied status", "ManifestWorkName", manifestWorkName)
 
-		if *manifestWork.Status.ResourceStatus.Manifests[0].StatusFeedbacks.Values[0].Value.String != string(ocsv1.StorageClusterPeerStatePeered) {
-			logger.Info("StorageClusterPeer has not reached Peered status", "ManifestWorkName", manifestWorkName)
+		if len(manifestWork.Status.ResourceStatus.Manifests) > 0 {
+			if *manifestWork.Status.ResourceStatus.Manifests[0].StatusFeedbacks.Values[0].Value.String != string(ocsv1.StorageClusterPeerStatePeered) {
```

Comment:
```suggestion
                mwManifests := manifestWork.Status.ResourceStatus.Manifests
		if len(mwManifests) > 0 && mwManifests[0].StatusFeedbacks.Values[0].Value.String != string(ocsv1.StorageClusterPeerStatePeered) {
```
And maybe a better variable name mwManifests? Could there also be a chance that StatusFeedbacks.Values are not populated, and we might get a panic here as well.